shapely/constructive.py

```diff
@@ -914,6 +927,11 @@ def snap(geometry, reference, tolerance, **kwargs):
     <POLYGON ((0 0, 0 10, 8 10, 10 0, 0 0))>
     >>> snap(polygon, LineString([(8, 10), (8, 0)]), tolerance=5)
     <POLYGON ((0 0, 0 10, 8 10, 8 0, 0 0))>
+    >>> square = Polygon([(1,1), (2, 1), (2, 2), (1, 2), (1, 1)])
+    >>> line = LineString([(0,0), (0.8, 0.8), (1.8, 0.95), (2.6, 0.5)])
+    >>> result = snap(line, square, 0.5)
+    >>> result.wkt
+    'LINESTRING (0 0, 1 1, 2 1, 2.6 0.5)'
```

Comment:
I know you copied this from the existing docstrings, but do we think this one adds value to the other examples that are already present? 
If not, I think we can also just leave it out.

Reply:
Maybe these are more helpful examples to add instead:
https://postgis.net/docs/ST_Snap.html

Reply:
how about now?